.travis.yml

```diff
@@ -0,0 +1,21 @@
+sudo: required
+
+language: java
+
+services:
+  - docker
+
+before_install:
+  - wget http://mirrors.ae-online.de/apache/maven/maven-3/3.3.9/binaries/apache-maven-3.3.9-bin.tar.gz
+  - tar xf apache-maven-3.3.9-bin.tar.gz
+  - export M2_HOME=$PWD/apache-maven-3.3.9
+  - export PATH=$M2_HOME/bin:$PATH
```

Comment:
According to the [guide](https://docs.travis-ci.com/user/languages/java/#Overview) it comes with Maven 3.2. If we really have to use higher version I would rather recommend using [Maven Wrapper](https://github.com/takari/maven-wrapper) instead and put it as a part of the repository. [That's how you can configure travis](https://github.com/arquillian/arquillian-recorder/blob/master/.travis.yml#L10) to use it.

Most likely you would have to add wrapper `jar` using `git add --force .mvn/wrapper/maven-wrapper.jar` because `jar`s are ignored by default.